src/Middleware/HttpLogging/test/HttpLoggingMiddlewareTests.cs

```diff
@@ -86,6 +86,7 @@ public async Task NoopWhenLoggingDisabled()
         Assert.DoesNotContain(TestSink.Writes, w => w.Message.Contains("Connection: keep-alive"));
         Assert.DoesNotContain(TestSink.Writes, w => w.Message.Contains("Body: test"));
         Assert.DoesNotContain(TestSink.Writes, w => w.Message.Contains("StatusCode: 200"));
+        Assert.DoesNotContain(TestSink.Writes, w => w.Message.Contains("Response:"));
```

Comment:
Can we just assert there are no logs with the `Microsoft.AspNetCore.HttpLogging.HttpLoggingMiddleware` category name?

Reply:
Done